rules/integrations/endpoint/elastic_endpoint_security.toml

```diff
@@ -17,7 +19,22 @@ index = ["logs-endpoint.alerts-*"]
 language = "kuery"
 license = "Elastic License v2"
 max_signals = 10000
-name = "Endpoint Security"
+name = "Elastic Defend"
```

Comment:
Please hold off renaming this rule till we change the text referencing it in the Elastic defend policy page.